### DISCUSSION TITLE
Table Row Focus with Embedded Button

### DISCUSSION THREAD

**Discussion by qscheitlin** [UI5/webcomponents#7145] [Q&A]
Hello, 

When I select a button within a currently selected row, it appears like the "focus" briefly changes (like the row is being reselected). I'm wondering if this is expected behavior because I'm not really selecting a row in this scenario, but rather the button within the row.  It also doesn't appear like any other events are being triggered in response to the button click.  Table mode is "SingleSelect" and TableRow type is "Inactive".

<img width="468" alt="image" src="https://github.com/SAP/ui5-webcomponents-react/assets/127273220/4cfdb220-cfbf-48fa-b149-34077b0f24f4"> --> before selecting the edit button

<img width="468" alt="image" src="https://github.com/SAP/ui5-webcomponents-react/assets/127273220/48cb7410-5b7e-4ff8-9473-2eb72e270c51"> --> while selecting the edit button

**Comment by Lukas742** [comment]
Hi @qscheitlin 

I'll forward this question to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by petyabegovska** [comment]
Hi @qscheitlin,

Can you please open an issue from this discussion and provide an issolated example  if possible by forking [this codesandbox](https://codesandbox.io/s/71r1x5o51q?fontsize=14&module=%2Findex.html).

Best regards,
Petya

**Comment by qscheitlin** [comment]
Hi @petyabegovska, absolutely.  I'll create an isolated example by EOW and then will create an issue from this discussion.  Thanks for you help @Lukas742 and @petyabegovska.
